HopsanGUI/HcomHandler.cpp

```diff
@@ -8018,7 +8023,12 @@ bool HcomHandler::evaluateArithmeticExpression(QString cmd)
             return false;
         }
 
-        if(mAnsType==Scalar)
+        if(mAnsType == Wildcard && !pars.isEmpty())
+        {
```

Comment:
Also here, I assume that this is some sort of assignment to parameters like Gain.p = Gain2.p